tests/conftest.py

```diff
@@ -29,10 +36,169 @@ def pytest_collection_modifyitems(config, items):
     )
     if not gpu_device_available:
         skip_gpu = pytest.mark.skip(reason="No devices available")
+        skip_bm = pytest.mark.skip(reason="Benchmarking disabled")
```

Comment:
so GPU is required for running the benchmarks, why?

Reply:
This was a bug. Move it out of the if. It just should skip all bm tests in normal testing mode